docs/source/tutorials/token_classification.ipynb

```diff
@@ -0,0 +1,730 @@
+    "        given_words.extend(words) \n",
+    "        given_labels.extend(label)\n",
+    "\n",
+    "    sentences = list(map(get_sentence, given_words)) \n",
+    "\n",
+    "    sentences, mask = filter_sentence(sentences) \n",
+    "    given_words = [words for m, words in zip(mask, given_words) if m] \n",
+    "    given_labels = [labels for m, labels in zip(mask, given_labels) if m] \n",
+    "\n",
+    "    maps = [0, 1, 1, 2, 2, 3, 3, 4, 4] \n",
+    "    given_labels = [mapping(labels, maps) for labels in given_labels] </details>"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 4,
+   "id": "f632872e",
+   "metadata": {
+    "nbsphinx": "hidden"
+   },
+   "outputs": [],
+   "source": [
+    "filepaths = ['data/train.txt', 'data/valid.txt', 'data/test.txt'] \n",
+    "given_entities = ['O', 'B-MISC', 'I-MISC', 'B-PER', 'I-PER', 'B-ORG', 'I-ORG', 'B-LOC', 'I-LOC'] \n",
+    "merged_entities = ['O', 'MISC', 'PER', 'ORG', 'LOC'] \n",
+    "entity_map = {entity: i for i, entity in enumerate(given_entities)} \n",
+    "\n",
+    "def readfile(filepath, sep=' '): \n",
+    "    lines = open(filepath)\n",
+    "    \n",
+    "    data, sentence, label = [], [], []\n",
+    "    for line in lines:\n",
+    "        if len(line) == 0 or line.startswith('-DOCSTART') or line[0] == '\\n':\n",
+    "            if len(sentence) > 0:\n",
+    "                data.append((sentence, label))\n",
+    "                sentence, label = [], []\n",
+    "            continue\n",
+    "        splits = line.split(sep) \n",
+    "        word = splits[0]\n",
+    "        if len(word) > 0 and word[0].isalpha() and word.isupper():\n",
+    "            word = word[0] + word[1:].lower()\n",
+    "        sentence.append(word)\n",
+    "        label.append(entity_map[splits[-1][:-1]])\n",
+    "\n",
+    "    if len(sentence) > 0:\n",
+    "        data.append((sentence, label))\n",
+    "        \n",
+    "    given_words = [d[0] for d in data] \n",
+    "    given_labels = [d[1] for d in data] \n",
+    "    \n",
+    "    return given_words, given_labels \n",
+    "\n",
+    "given_words, given_labels = [], [] \n",
+    "\n",
+    "for filepath in filepaths: \n",
+    "    words, label = readfile(filepath) \n",
+    "    given_words.extend(words) \n",
+    "    given_labels.extend(label)\n",
+    "    \n",
+    "sentences = list(map(get_sentence, given_words)) \n",
+    "\n",
+    "sentences, mask = filter_sentence(sentences) \n",
+    "given_words = [words for m, words in zip(mask, given_words) if m] \n",
+    "given_labels = [labels for m, labels in zip(mask, given_labels) if m] \n",
+    "\n",
+    "maps = [0, 1, 1, 2, 2, 3, 3, 4, 4] \n",
+    "labels = [mapping(labels, maps) for labels in given_labels] "
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "cd956e6c",
+   "metadata": {},
+   "source": [
+    "Here, `issue` is a list of tuples `(i, j)`, which corresponds to the `j`'th token of the `i`'th sentence. "
```

Comment:
`issues` not `issue`

Reply:
Add another sentence:

These are the tokens cleanlab thinks may be  badly labeled in your dataset.